src/helpers/normalizeEvent.js

```diff
@@ -19,7 +19,7 @@ const normalizedEvent = event => {
     }
     return JSON.parse(event.Records[0].body);
   } else if ("httpMethod" in event) {
-    return JSON.parse(event.body);
+    return JSON.parse(event.queryStringParameters.json);
```

Comment:
I see your point. My REST senses tell me this shouldn't be a GET request (since it has side-effect), so I would really like to keep my POSTs :) 

But I'm no REST police, I see how it might make sense to "get" a screenshot, so I think it would be nice to support both cases (body for POSTs, qs for GETs). 

What do you think?

Reply:
After thinking about it more, I agree with you. A POST is best since a capture is being created. I added an additional check to clarify POST support and error out on GET support since API Gateway doesn't support GET request body.